src/utils.js

```diff
@@ -4,3 +4,41 @@ module.exports.generateRandomKey = function (len) {
   len = len || 10;
   return Math.random().toString(36).substring(2).substring(0, len);
 };
+
+module.exports.deepGet = function(collection, path){
```

Comment:
All of these changes are probably not needed if keyPath string support is dropped?